<launch>
  <!-- Publish robot model and its joints -->
  <arg name="model" />
  <param name="robot_description" command="$(find xacro)/xacro --check-order $(arg model)" />
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    <rosparam param="source_list">["move_joints"]</rosparam>
  </node>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
</launch>